platform-configs/include/default.robot

```diff
@@ -24,6 +24,7 @@ ${FLASH_VERIFY_METHOD}=                             ${EMPTY}
 ${WIFI_CARD}=                                       ${EMPTY}
 ${MAX_CPU_TEMP}=                                    ${EMPTY}
 ${INTERNAL_PROGRAMMER_CHIPNAME}=                    Opaque flash chip
+${MIN_ACCEPTED_NET_SPEED}=                          2.32
```

Comment:
It shouldn't be here. Some platform have only 1Gb Ethernet ports and some have 2.5Gb Ethernet. So this value will never be reached by some platforms...

Reply:
This value is just the max link speed of a device minus TCP overhead, so we can get this programatically:

```
echo $(($(cat /sys/class/net/enp0s13f0u1u2c2/speed) * 0.93 / 1000))
```

Reply:
Didn't use this way because contents of this file for disabled nic is just "-1". But I have used the ```ethtool```. You can check my solution at: fca76a3e13ec001b787f9a90a25930eda2547f2d